lib/bot.rb

```diff
@@ -0,0 +1,19 @@
+class Bot
+  def initialize(client: nil)
+    @client = client || Slack::Web::Client.new
```

Comment:
yeah the original intention here is to make it possible to inject a fake client in a test, but you're right, I think we won't need to, because we'll be able to use the fake bot wrapper instead.

Reply:
Yeah, I would either write the test or delete this and create an issue "write a test for that".
Otherwise, the risk is just to forget to do anything about it and only have this bit of code here as a clue.

Reply:
I removed this, I think it's enough to spy on the send method on Bot, which is only a thin object wrapper around the Slack client now.